### PR TITLE
Support PyTorch 1.10

### PR DESCRIPTION
**Pull request by mravanelli**
I did some tests and everything seems to work. There is only the need to fix some warnings.

### CODE REVIEW

**Comment by mravanelli** [comment]
I fixed some warnings. I think we can merge it and test it better the multi-gpu training and the torchaudio transducer loss separately.
